src/components/ConnectCompo/FilterBottomSlide.js

```diff
@@ -76,13 +78,36 @@ const FilterBottomSlide = (props) => {
 
 
 
+
     const [isCheckedList, setIsCheckedList] = useState([
       false,
       false,
       false,
       false,
       false, 
     ]);
+    
+    const [mbtiCnt, setMbtiCnt] = useState(0);
+    const [hobbyCnt, setHobbyCnt] = useState(0);
+    
+    useEffect(() => {
+      if (mbtiCnt >= 3) {
+        setIsDisabled(true);
+      } else {
+        setIsDisabled(false);
+      }
```

Comment:
```suggestion
setIsDiabled(mbtiCnt >= 3)
```

[참고] 한 줄로도 변경이 가능할 것 같네요!